includes/class-wc-product-variable.php

```diff
@@ -724,6 +724,33 @@ public static function sync_attributes( $product_id, $children = false ) {
 	}
 
 	/**
+	 * Does a child have a weight set?
+	 * @since 2.7.0
+	 * @return boolean
+	 */
+	public function child_has_weight() {
+		return boolval( get_post_meta( $this->id, '_child_has_weight', true ) );
```

Comment:
Can you remind me... Is our minimum PHP version 5.6 or just is that just the recommended minimum? (System status recommends 5.6, but I now WP itself still supports lower versions). Just wanted to bring it up since `boolval` is >= 5.5